docs/api-docs/getting-started/authentication.md

```diff
@@ -1,30 +1,90 @@
 
-## Customer Login API
+For more details, see [GraphQL API Authentication](/api-docs/storefront/graphql/graphql-storefront-api-overview#authentication) and [Obtaining Store API Credentials](/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials).
 
-The Customer Login API requires authentication via a [JWT token](https://jwt.io/) and your app's OAuth client ID.
+```http title="GraphQL authentication header"
+Authorization: Bearer {{generated_jwt}}
+```
 
-For details, see [Customer Login API](/api-docs/customers/customer-login-api).
+## Class III: App APIs
 
-## Current Customer API
+Apps add novel ways of using BigCommerce store data and can connect third-party services to merchant stores. Apps authenticate with JWT access tokens. Create an app OAuth API account to enable your app to request and manage BigCommerce-generated JWTs on behalf of merchant stores. You can change the scope of an app OAuth API account at any time. When you modify the OAuth scope, BigCommerce will invalidate the API account's existing JWT access tokens. To generate new access tokens, BigCommerce will prompt your users to reauthorize the app so that they can accept the new OAuth scope.
 
-Your application's client ID must be included in the Current Customer API request to receive a response. For details, see [Current Customer API](/api-docs/customers/current-customer-api).
+For more on working with apps, see our [Guide to Building Apps](/api-docs/apps/guide/intro). The sections on [Implementing OAuth](/api-docs/apps/guide/auth) and [Callback Handlers](/api-docs/apps/guide/callbacks) are particularly relevant.
+
+```http title="App authentication header for store management APIs"
+X-Auth-Token: {{jwt_access_token}}
+```
+
+### Shipping Provider API
+
+The Shipping Provider API connects a shipping service to stores and supports a range of shipping-related actions, such as custom rate table calculations. You must create or work within an app to use the Shipping Provider API. Create an app OAuth API account to use it.
+
+Learn more about working with the [Shipping Provider API](/api-docs/providers/shipping).
+
+```http title="App authentication header for the Shipping Provider API"
+X-Auth-Token: {{jwt_access_token}}
+```
+
+### Tax Provider API
+
+The Tax Provider API connects tax calculation and filing services to stores. You must create or work within an app to use the Tax Provider API. Create an app OAuth API account to use it.
+
+Learn more about working with the [Tax Provider API](/api-docs/providers/tax).
+
+```http title="App authentication header for the Tax Provider API"
+X-Auth-Token: {{jwt_access_token}}

```

Comment:
Same question here - does the Tax Provider API really use different auth?

Reply:
sounds like not.